### PR TITLE
Bootstrap PWA

### PR DESCRIPTION
**Pull request by luistorres**
Adds basic PWA functionalities, equivalent to create-react-app. This is wip, but allows someone to pick it up easily in the future.

### CODE REVIEW

File: src/frontend/public/offline.html [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+    <link rel="manifest" href="./manifest.json" />
+    <meta name="theme-color" content="#DE463E" />
+  </head>
+  <body>
+    <header class="header">
+      BVE offline
+    </header>
+    <main>
+      <
```
Comment (jfranciscosousa) [inline src/frontend/public/offline.html]: :rotating_light:

File: src/frontend/registerServiceWorker.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+// it offline capabilities. However, it also means that developers (and users)
+// will only see deployed updates on the "N+1" visit to a page, since previously
+// cached resources are updated in the background.
+
+// To learn more about the benefits of this model, read https://goo.gl/KwvDNy.
+// This link also includes instructions on opting out of this behavior.
+
+export default function register() {
+  // Register the service worker
+  // if (process.env.NODE_ENV === "production" && "serviceWorker" in navigator) {
```
Comment (jfranciscosousa) [inline src/frontend/registerServiceWorker.js]: Wanna keep these comments?

**Review by jfranciscosousa — CHANGES_REQUESTED** [review]

File: src/frontend/registerServiceWorker.js (hunk 2 above)
Comment (luistorres) [reply src/frontend/registerServiceWorker.js]: Fixed doctor

File: src/frontend/public/offline.html (hunk 1 above)
Comment (luistorres) [reply src/frontend/public/offline.html]: thanks

**Review by jfranciscosousa — APPROVED** [review]

**Pull request closed without merging**